src/components/pod/pod.component.js

```diff
@@ -0,0 +1,340 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import I18n from 'i18n-js';
+import Event from '../../utils/helpers/events/events';
+import tagComponent from '../../utils/helpers/tags/tags';
+import {
+  StyledBlock,
+  StyledCollapsibleContent,
+  StyledContent,
+  StyledDescription,
+  StyledEditAction,
+  StyledEditContainer,
+  StyledFooter,
+  StyledPod,
+  StyledHeader,
+  StyledSubtitle,
+  StyledTitle,
+  StyledArrow
+} from './pod.style.js';
+
+class Pod extends React.Component {
+  state = {
+    isCollapsed: this.props.collapsed,
+    isHovered: false,
+    isFocused: false
+  };
+
+  // eslint-disable-next-line camelcase
+  UNSAFE_componentWillReceiveProps() {
```

Comment:
Maybe its worth scraping deprecated UNSAFE_componentWillReceiveProps and doing it as demonstrated here:
https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html ?

Reply:
removed and replaced by direct change of the state in the "processPodEditEvent" method